socket/packet/server_list_response.go

```diff
@@ -0,0 +1,59 @@
+package packet
+
+import (
+	"github.com/sandertv/gophertunnel/minecraft/protocol"
+)
+
+// ServerListResponse is sent by the proxy in response to ServerListResponse. It sends list of all
+// the servers connected to the proxy (including offline servers)
+type ServerListResponse struct {
+	// Servers represents all the servers connected to the proxy
+	Servers []ServerEntry
+}
+
+type ServerEntry struct {
+	// Name is name of the server
+	Name        string
+	// Group is group of the server
+	Group       string
+	// IsOnline returns if the server is currently online
+	IsOnline    bool
+	// PlayerCount is count of online players connected to that server through proxy
+	PlayerCount uint16
+}
+
+// ID ...
+func (*ServerListResponse) ID() uint16 {
+	return IDServerListResponse
+}
+
+// Marshal ...
+func (pk *ServerListResponse) Marshal(w *protocol.Writer) {
+	l := uint16(len(pk.Servers))
```

Comment:
Please use a uint32 for the length rather than a uint16

Reply:
You think someone would register more than 65535 servers? 🤔

Reply:
Even if they wouldn't, it's still better to have consistency in the protocol